### ISSUE TITLE
Facture d'export PDF > couleur absente

### ISSUE THREAD

**Issue by Playologie** [thibaultvanc/playologie_exports#61]
La couleur "grey viscose" du produit Long sleeve Charisme est absente du PDF de BL et de Facture d'export alors qu'elle est bien présente à l'écran

**Comment by organitdev** [comment]
Pouvez - vous préciser la commande  concernée ?

**Comment by Playologie** [comment]
Préprod 
ManFashionBrand (mfb@playologie.com / man2017)

Commande MFB-070317-009 (http://preprod3.playologie.com/brand-order/#/order/3646)
BL SHIP-070317-006 (http://preprod3.playologie.com/brand-order/#/shipment/21)
Facture Export EXP-030417-003 (à télécharger depuis le BL)
> Long sleeve Charisme : grey viscose OK

Commande MFB-280217-004 (http://preprod3.playologie.com/brand-order/#/order/3639)
BL SHIP-080317-007 (http://preprod3.playologie.com/brand-order/#/shipment/22) 
Facture Export EXP-080317-001 (à télécharger depuis le BL)
> Long sleeve Charisme : grey viscose pas présent

L'un est un imprimé, l'autre une couleur

**Comment by thibaultvanc** [comment]
voici le pdf de la commande  MFB-280217-04

http://preprod3.playologie.com/export/order/f8384beccaefc119763b0cd2f54b22be.pdf

je ne vois pas le pb. elle est conforme à se qui est sur le web

N'hésitez pas à m'appeler

**Comment by Playologie** [comment]
non, dans commande MFB-280217-004, l'article "Long sleeve Charisme" contient 2 couleurs.
Dans le BL aussi, mais pas dans la facture d'export (manque 1 couleur)

**Comment by thibaultvanc** [comment]
besoin de voir avec vous svp
